Quadriga/src/main/java/edu/asu/spring/quadriga/service/textfile/impl/TextFileManager.java

```diff
@@ -114,7 +120,16 @@ public String retrieveTextFileContent(String txtId) throws FileStorageException
         if (!fileName.contains("."))
             fileName += ".txt";
         return fileSaveServ.retrieveFileFromLocal(fileName, txtId);
-
     }
 
+    @Override
+    public ITextFile updateTextFileAccessibility(ITextFile textFile) {
```

Comment:
Don't do this here. why don't you just set the accessibility where ever you call this method and then just save it? otherwise, what if we add another accessibility options (e.g. for authorized users only)?